src/lib.rs

```diff
@@ -1,3 +1,27 @@
+/*!
+The `windows-app` crate makes the [Windows App SDK](https://github.com/microsoft/WindowsAppSDK) available to Rust developers making it possible
+to create modern Windows app in pure Rust.
```

Comment:
Nit: Tweaked to make the sentence timeless and increase clarity.

```suggestion
The `windows-app` crate makes the [Windows App SDK](https://github.com/microsoft/WindowsAppSDK) available to Rust developers making it possible
to create Windows apps in pure Rust.
```